pkg/terminal/out.go

```diff
@@ -0,0 +1,220 @@
+package terminal
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/go-delve/delve/pkg/terminal/colorize"
+	"github.com/mattn/go-isatty"
+)
+
+// transcriptWriter writes to a pagingWriter and also, optionally, to a
+// buffered file.
+type transcriptWriter struct {
+	fileOnly     bool
+	w            *pagingWriter
```

Comment:
nit: can we change this to `pager` or `pw` just to remove some of the stutter of `w.w.<foo>`.

Reply:
Done.